omni_blocks/templates/blocks/pull_quote_block.html

```diff
@@ -0,0 +1 @@
+<blockquote class="pquote">{{ value|safe }}</blockquote>
```

Comment:
Could we have it like this please, to save time overriding the template for every site?

```
<blockquote class="pullquote">
    <div class="blockquote__inner">
        <p>{{ value|safe }}</p>
    </div>
</blockquote>
```